.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: python
 python:
     - 2.7
-    - 3.4
     - 3.5
     - 3.6
+    - 3.7
+    - 3.8-dev
```

Comment:
Looks like we have to fix up the tox logic:
https://travis-ci.org/gentoo/portage/jobs/597895462
```
0.21s$ if [[ ${TRAVIS_PYTHON_VERSION} == ?.? ]]; then tox -e py${TRAVIS_PYTHON_VERSION/./}; else tox -e ${TRAVIS_PYTHON_VERSION}; fi
ERROR: unknown environment '3.8-dev'
```